Using dir-babel results in Error: Cannot find module 'mkdirp'

It looks like the mkdirp dependency is missing.

```
Error: Cannot find module 'mkdirp'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (.../node_modules/dir-babel/lib/dir-babel.js:20:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
```

--- a/package.json
+++ b/package.json
@@ -28,6 +28,7 @@
     "babel-core": "^6.24.1",
     "batch-promises": "0.0.3",
     "glob": "^7.1.1",
+    "mkdirp": "^0.5.1",
     "yargs": ">=7.0.1"
   },
   "keywords": [

